finetune/lm_base.py

```diff
@@ -367,6 +373,28 @@ def _define_placeholders(self):
         self.M = tf.placeholder(tf.float32, [None, self.max_length])  # sequence mask
         self.Y = tf.placeholder(tf.float32, [None, self.target_dim])  # classification targets
 
+    def lm_predict(self, max_length=None, seed_text=''):
+        """
+        Performs a prediction on the Language modeling objective given some seed text. It uses a noisy greedy decoding.
```

Comment:
Would change this wording up (predicting on an objective sounds odd) and then go ahead and merge.